src/components/SystemdServiceFileGenerator/index.jsx

```diff
@@ -84,14 +84,12 @@ ExecStart=${formData.nodeBinPath} \\
           run \\ 
           --name ${formData.nodeName} \\
           --base-path ${formData.nodeData} \\
-          --state-pruning archive-canonical \\
-          --keep-blocks ${formData.archival === "on" ? "archive-canonical" : "256"} \\
           --chain ${chain} \\
           --farmer \\
           --listen-on /ip4/0.0.0.0/tcp/${formData.nodePort} \\
           --dsn-listen-on /ip4/0.0.0.0/udp/${formData.nodeDsnPort}/quic-v1 \\
-          --dsn-listen-on /ip4/0.0.0.0/tcp/${formData.nodeDsnPort} \\
-          ${formData.extraNodeArgs ? "\\\n" + formData.extraNodeArgs : ""}
+          --dsn-listen-on /ip4/0.0.0.0/tcp/${formData.nodeDsnPort} ${formData.extraNodeArgs === "" ? "" : "\\\n"}\
```

Comment:
Your suggestion will not work. That will always add the trailing `\` regardless of whether there are `extraNodeArgs` set which is what we are trying to avoid.

Reply:
I see, in that case let's just use a new line for the last line and add `\\` after `--dsn-listen-on /ip4/0.0.0.0/tcp/${formData.nodeDsnPort}`